whitelists/facebook.json

```diff
@@ -26,4 +26,5 @@
   "bitkubicon.kmitl",
   "bitkubicon.bru",
   "bitkubicon.cmu"
```

Comment:
Forgot a comma, this would crash the system because it's not formatted correctly